.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+install: skip
+
+os: linux
+dist: focal
+jdk: oraclejdk8
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+
+script:
+  - mkdir -p ./google-play-billing/libs
+  - wget -O ./google-play-billing/libs/godot-libs.release.aar https://downloads.tuxfamily.org/godotengine/3.2.2/rc2/godot-lib.3.2.2.rc2.release.aar
```

Comment:
Can you match the name of the downloaded file.

Reply:
Done